src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import './TodoForm.css';
+
+const TodoForm = (props) => {
+  return (
+    <form className="todo-form" onSubmit={props.addTodoItem}>
+      <div className="submit-form">
+        <input
+          type="text"
+          name="newTodoItem"
+          placeholder="What todo?"
+          value={props.text}
+          onChange={props.changeHandler}
+        />
+        <button type="submit" onClick={props.addTodoItem}>
```

Comment:
this button wouldn't need an onClick because it is nested inside the Form tag therefore the button will automatically be bind to the onSubmit attribute on the Form element when being click on this button or hitting the enter key after typing on the input . All because they are nested inside the form tag